packages/shared/lib/core/error/constants/api-error-parameters.constant.ts

```diff
@@ -0,0 +1,21 @@
+import { ClientError, ErrorFromApi } from '../enums'
+import { IErrorParameters } from '../interfaces'
+
+export const API_ERROR_PARAMETERS: Readonly<
+    { [key in ErrorFromApi]?: { [key in ClientError]?: Partial<IErrorParameters> } }
```

Comment:
i think this type isn't correct.
If the toplevel key is `ErrorFromApi.ClientError` then the subkeys can be of type `ClientError`.
But if the toplevel key is e.g. `ErrorFromApi.InsufficientFundsError`  (which you get from walletrs as `{type: "InsufficientFunds", message: "insufficient amount of funds ..."}`, we wouldn't add these subtypes in the `ClientError` enum